kill.c

```diff
@@ -73,6 +73,12 @@ int kill_wait(const poll_loop_args_t* args, pid_t pid, int sig)
     }
     meminfo_t m = { 0 };
     const unsigned poll_ms = 100;
+    if (args->kill_process_group) {
+        if ((pid = getpgid(pid)) < 0) {
+            fatal(7, "%s: could not get PGID: %s", __func__, strerror(errno));
```

Comment:
fatal error is too much here ;)

getpgid can fail when the process has already exited, this is normal. Please just "return res" as below.

Reply:
That sounds more reasonable indeed. Thanks!